test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -28,4 +28,6 @@ jest.mock('@folio/stripes/components', () => ({
     <div ref={ref} {...rest}>{children}</div>
   )),
   PaneMenu: jest.fn((props) => <div>{props.children}</div>),
+  Row: jest.fn(({ children, ...rest }) => <div {...rest}>{children}</div>),
+  Col: jest.fn((props) => <div>{props.children}</div>),
```

Comment:
Lets keep alphabetical order of stripes components here

Reply:
refactored